blog/2025-07-29-latest-benefits-of-rmq-and-migrating-to-qq-along-the-way/index.md

```diff
@@ -0,0 +1,380 @@
+---
+title: "Blue-Green upgrade to migrate from Classic Mirrored Queues to Quorum Queues"
```

Comment:
This is the nittiest nitpick, but the docs pretty consistently use the term "Mirrored Classic Queues" -

https://www.google.com/search?q=site%3Arabbitmq.com+%22mirrored+classic+queues%22

...as much as we're all used to saying CMQs.